src/dataset-builder/DatasetBuilderUtils.ts

```diff
@@ -309,3 +309,7 @@ export const HighlightConceptName = ({ conceptName, searchFilter }): ReactElemen
     span([conceptName.substring(endIndex)]),
   ]);
 };
+
+export const displayParticipantCount = (count: number): string => {
+  return count <= 19 && count > 0 ? 'Less than 20' : count.toString();
```

Comment:
Based off of Sky's suggestions of changing the signature to return a `ReactElement`
This is how you'd change from returning a string to a ReactElement.
`return div([count <= 19 && count > 0 ? 'Less than 20' : count.toString()]);`

Then, anywhere `displayParticipantCount` was called you're going to replace with 
`DisplayParticipantCount( { count: anyNumberValue })` 

Example: CohortEditor on line 432:
From:
`? displayParticipantCount(groupParticipantCount.state.result.total)`

To:
`? DisplayParticipantCount({ count: groupParticipantCount.state.result.total })`